template.js

```diff
@@ -1,18 +1,69 @@
+/* eslint-disable no-undef, prefer-template, no-useless-escape */
 const readme = require('@readme/cloudflare-worker');
 
 addEventListener('fetch', event => {
   event.passThroughOnException();
 
-  event.respondWith(respond(event));
+  if (determineRouting(event.request.url)) {
+    event.respondWith(respond(event));
+  }  else {
+    event.respondWith(fetch(event.request));
+  }
 });
 
 async function respond(event) {
   const { response, har } = await readme.fetchAndCollect(event.request);
 
-  event.waitUntil(readme.metrics('API_KEY', {
+  event.waitUntil(readme.metrics(INSTALL_OPTIONS.API_KEY, {
     id: response.headers.get('x-readme-id'),
     label: response.headers.get('x-readme-label'),
   }, event.request, har));
 
   return response;
 }
+
+function determineRouting(url) {
```

Comment:
Could we put `determineRouting` into a separate file and test it against all of the examples in https://developers.cloudflare.com/workers/api/route-matching/? I get really anxious about URL matching features in general haha

Reply:
Yeah that sounds like a good idea! I can do that.

Reply:
Done :)